src/components/entity/list.vue

```diff
@@ -572,12 +574,17 @@ export default {
       this.odataEntities.value?.forEach(e => { e.__system.selected = false; });
       this.allSelected = false;
     },
+    cancelBackgroundRefresh() {
+      this.odataEntities.cancelRequest();
+      this.deletedEntityCount.cancelRequest();
```

Comment:
(Just FYI, no change needed.) Initially I thought, maybe there's no need to cancel this request. It doesn't affect much. But then I remembered that the bulk operations mutate `deletedEntityCount.value`. There could be a race condition if this request weren't canceled. So I think the behavior is clearer this way. :+1: